src/test/java/teammates/test/cases/action/InstructorFeedbackAddActionTest.java

```diff
@@ -219,11 +223,13 @@ protected InstructorFeedbackAddAction getAction(String... params) {
         return (InstructorFeedbackAddAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
-    protected String getPageResultDestination(
-            String parentUri, String courseId, String fsname, String userId, boolean isError) {
+    protected String getPageResultDestination(String parentUri, String courseId, String fsname,
+            boolean isToBeLoadedInEditMode, String userId, boolean isError) {
```

Comment:
@whipermr5 To make this and similar changes, I would need to overload the method being called in [this line](https://github.com/TEAMMATES/teammates/pull/7615/files#diff-962b47c7f33822774b4debc9ddca5a16R82) (Ie `getInstructorFeedbackLink`) to also accept a set of params that does not include the `isToBeLoadedInEditMode` boolean. Would overloading be the best approach here? Or should I use the `boolean... shouldLoadInEditMode` approach as an optional param and then handle the null case using `if (shouldLoadInEditMode[0] == null`) ?